Live_data_update.sql

```diff
@@ -288,3 +289,45 @@ update cqc."Establishment" set "NameValue"='Stellar Healthcare Solutions Limited
 select "EstablishmentID","NameValue","LocationID" from cqc."Establishment" where "EstablishmentID"=2399;
 
 
+
+--------------14 June 2019
+---https://trello.com/c/StXL6vqb
+
+select "EstablishmentID","MainServiceFKValue","LocationID" from cqc."Establishment" where "EstablishmentID"=2296;
+
+Update cqc."Establishment" set "LocationID"='1-6487895351' where "EstablishmentID"=2296;
+
+select "EstablishmentID","MainServiceFKValue","LocationID" from cqc."Establishment" where "EstablishmentID"=2296;
+
+
+----- 17 June 2019
+
+
+-----------------            https://trello.com/c/StXL6vqb
+
+select "EstablishmentID","MainServiceFKValue","LocationID" from cqc."Establishment" where "EstablishmentID"=2296;
+
+Update cqc."Establishment" set "LocationID"='1-6487895351' where "EstablishmentID"=2296;
```

Comment:
this is a duplicate of the above dated 14th June 2019.

Reply:
Hi Warren, 
Yes that's true as Jason updated same card and wanted to update the locationid and email.
Email he sent I found in database so didn't put here in the change request.

Many Thanks
Shakir